deploy/helm/edot-collector/kube-stack/README.md

```diff
@@ -0,0 +1,63 @@
+
+The chart is installed with a provided default `values.yaml` file that can be customized when needed.
+
+### DaemonSet collectors
+
+The OpenTelemetry components deployed within the DaemonSet EDOT collectors are responsible for observing specific signals from each node. To ensure complete data collection, these components must be deployed on every node in the cluster. Failing to do so will result in partial and potentially incomplete data.
+
+The DaemonSet collectors handle the following data:
+
+- Host Metrics: Collects host metrics (hostmetrics receiver) specific to each node.
+- Kubernetes Metrics: Captures metrics related to the Kubernetes infrastructure on each node.
+- Logs: Utilizes a filelog receiver to gather logs from all Pods running on the respective node.
+- OTLP Traces Receiver: Opens an HTTP and a GRPC port on the node to receive OTLP trace data.
+
+### Deployment collector
+
+The OpenTelemetry components deployed within a Deployment collector focus on gathering data at the cluster level rather than at individual nodes. Unlike DaemonSet collectors, which need to be deployed on every node, a Deployment collector operates as a standalone instance.
+
+The Deployment collector handles the following data:
+
+- Kubernetes Events: Monitors and collects events occurring across the entire Kubernetes cluster.
+- Cluster Metrics: Captures metrics that provide insights into the overall health and performance of the Kubernetes cluster.
+
+### Auto-instrumentation
+
+The Helm Chart is configured to enable zero-code instrumentation using the [Operator's Instrumentation resource](https://github.com/open-telemetry/opentelemetry-operator/?tab=readme-ov-file#opentelemetry-auto-instrumentation-injection) for the following programming languages:
+
+- Go
+- Java
+- Node.js
+- Python
+- .NET
+
+
+### Installation
+
+1. Create the `opentelemetry-operator-system` Kubernetes namespace:
+```
+$ kubectl create namespace opentelemetry-operator-system
+```
+
+2. Create a secret in Kubernetes with the following command.
+   ```
+   kubectl create -n opentelemetry-operator-system secret generic elastic-secret-otel \
+     --from-literal=elastic_endpoint='YOUR_ELASTICSEARCH_ENDPOINT' \
+     --from-literal=elastic_api_key='YOUR_ELASTICSEARCH_API_KEY'
+   ```
+   Don't forget to replace
+   - `YOUR_ELASTICSEARCH_ENDPOINT`: your Elasticsearch endpoint (*with* `https://` prefix example: `https://1234567.us-west2.gcp.elastic-cloud.com:443`).
+   - `YOUR_ELASTICSEARCH_API_KEY`: your Elasticsearch API Key
+
+3. Execute the following commands to deploy the Helm Chart.
+
+```
+$ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+$ helm repo update
+$ helm upgrade --install --namespace opentelemetry-operator-system opentelemetry-kube-stack open-telemetry/opentelemetry-kube-stack --values ./resources/kubernetes/operator/helm/values.yaml --version 0.3.0
```

Comment:
I would update the path to values.yaml relative to what will be in this folder
So not `./resources/kubernetes/operator/helm/values.yaml` rather /deploy/helm ....

Reply:
Good point, I think I would prefer to keep it relative to the README's directory: https://github.com/elastic/elastic-agent/pull/5822/commits/89cd2a68fc8c89651a352888ebe4cdc9d8cc562b